sdk/nodejs/kinesis/kinesisMixins.ts

```diff
@@ -151,7 +151,7 @@ function createFunctionFromEventHandler(
     if (handler instanceof Function) {
         return new lambda.CallbackFunction(name, {
             callback: handler,
-            policies: [iam.ManagedPolicy.AWSLambdaFullAccess, iam.ManagedPolicy.AWSLambdaKinesisExecutionRole]
+            policies: [iam.ManagedPolicy.AWSLambdaKinesisExecutionRole]
```

Comment:
This looks breaking since the role will have fewer permissions than previously. I think this should create a copy of `AWSLambdaFullAccess`, then attach that new policy plus the `AWSLambdaKinesisExecutionRole` policy here.

Reply:
No, the use of the LambdaFullAccess happens inside the CallbackFunction and each of SQS and Kinesis calls

Reply:
Right, but my point is that `CallbackFunction` should not indiscriminately create the `AWSLambdaFullAccess` policy. It should only be created when `policies` is not specified (as it currently works in the package).